src/lib/util/utils.ts

```diff
@@ -541,3 +541,43 @@ export function relaxModelLikeEntities(model: Model) {
   }
   return model
 }
+
+/**
+ * 
+ * @param jsonPath e.g paths['test!'].get.response
+ * @returns jonsPointer  /paths/test!/get/response
+ */
+
+export function pathToJsonPointer(jsonPath:string):string {
+  const replaceAllReg = (src:string):RegExp =>{
```

Comment:
Where is the source of this code? It's hard to read. Pls put some doc or comments here.